.konflux/manifests/components/knative-eventing-controller.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Component
+metadata:
+  annotations:
+    image.redhat.com/generate: "true"
+    appstudio.openshift.io/pac-provision: request
+  name: knative-eventing-controller
+spec:
+  componentName: knative-eventing-controller
+  application: serverless-operator-konflux
+  build-nudges-ref:
+    - serverless-bundle
+  source:
+    git:
+      url: https://github.com/openshift-knative/eventing.git
+      context: 
```

Comment:
Is the "empty" context for some debugging/documentation purposes? Or can this be removed.

Reply:
Probably can be empty. I'll remove it

Reply:
updated: b48d73d56981673e0897b38285cdc90a4909e391